src/_lib/modules/dropdowns/dropdown-menu.directive.ts

```diff
@@ -1,23 +1,43 @@
-import { Directive, HostListener, QueryList, AfterContentInit, ContentChildren, OnDestroy } from '@angular/core';
+import { Directive, HostListener, QueryList, AfterContentInit, ContentChildren, OnDestroy, ElementRef, ChangeDetectorRef } from '@angular/core';
 import { DropdownService } from "./dropdown.service";
 import { DropdownDirective } from "./dropdown.directive";
-import { map } from "rxjs/operators";
+import { filter, map } from "rxjs/operators";
 import { Subscription, Observable, merge } from 'rxjs';
 
 @Directive({
-  selector: '[nwDropdownMenu]',
-  exportAs: 'nw-dropdown-menu'
+    selector: '[nwDropdownMenu]',
+    exportAs: 'nw-dropdown-menu'
 })
 export class DropdownMenuDirective implements AfterContentInit, OnDestroy {
 
     @ContentChildren(DropdownDirective) nestedDropdowns: QueryList<DropdownDirective>;
 
     private _childrenVisibilityToggledSub: Subscription;
+    private _scrollSub: Subscription;
```

Comment:
In the interest of improvements, would you mind removing these subs and managing the observables with `takeUntil` instead?